docs/perf.md

```
@@ -0,0 +1,24 @@
+- Environment: Node.js v24.6.0 with the bundled `scripts/measure-content-init.js` harness.
+- Harness spins up a JSDOM instance of `https://x.com/home`, enables the `spw_dev` flag, and executes the target `content.js` five times.
+- Each run records `performance.now()` deltas from script injection to completion (including idle callbacks); timers/intervals are cleared after each run.
+- Commands used:
+  ```bash
+  git show HEAD:content.js > /tmp/content-baseline.js
+  node scripts/measure-content-init.js /tmp/content-baseline.js
+  node scripts/measure-content-init.js content.js
+  ```
+
+## Results
+| Build | Runs (ms) | Median | Average |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)